contrib/epee/include/net/abstract_http_client.h

```diff
@@ -0,0 +1,173 @@
+#pragma once
```

Comment:
This file has no copyright notice. Unfortunately, I think it will need to copy the notice at the top of the original file since this code dates back to the cryptonote fork.